app/views/research_sessions/focus.html.erb

```diff
@@ -1,17 +1,17 @@
 <%= render 'back_link' %>
 <div class="main-content">
   <%= render 'error_summary' %>
-  <h1 class="visually-hidden">Research Focus</h2>
```

Comment:
I *wondered* where that had come from. I suspect it's [this](https://github.com/barnardos/consent-form-builder-rails/pull/45/commits/13eba2025f9028a8e0f4f7e9a1cccd54720a8951)